ampbench_lib.js

```diff
@@ -775,10 +775,14 @@ function lib_fetch_cdn_validator_spec_file_revision(callback) {
  */
 function lib_load_validator(opt_force_reload) {
     let _force_reload = opt_force_reload || false;
-    const _validator = amphtml_validator.getInstanceFromFileSync(VALIDATOR_JS_FILE, _force_reload);
-    amphtml_validator_instance = _validator; // cache the module level instance here
-    amphtml_validator_spec_file_revision = lib_extract_validator_spec_file_revision(_validator.scriptContents);
```

Comment:
What is this used for?
The spec revision is just in the file by accident, it's not meant to be exported there and it's also not correct to assume that the same spec file revision gives the same behavior because the validator code can change as well.